src/ibex_bluesky_core/plans/reflectometry/_autoalign.py

```diff
@@ -82,7 +82,7 @@ def _optimise_axis_over_range(  # noqa: PLR0913 PLR0917
         dae (SimpleDae): A readable DAE object.
         alignment_param (AlignmentParam): The alignment parameter to be scanned over and optimised.
         fit_param (str): Which property of fit_method you aim to optimise. e.g centre (x0)
-            of a Gaussian. See fitting/fitting_utils for the possible options for each fitting
+            of a Gaussian. See fitting for the possible options for each fitting
```

Comment:
Feels like this is a find-and-replace gone a bit awry as this sentence:

> See fitting for the possible options for each fitting method.

doesn't really read quite right any more...